practices/graded-browser-support.md

```diff
@@ -0,0 +1,52 @@
+# Graded Browser Support
+
+
+## Our criteria for browser support
+
+
+We followe the guidelines for graded browser support outlined by Nate Koechley at Yahoo:
+[https://github.com/yui/yui3/wiki/Graded-Browser-Support](Yahoo Grade Browser Support guide)
```

Comment:
This applies to all the links actually. I think it's because the format should be [name](url)